src/webapp/azext_webapp/__init__.py

```diff
@@ -29,6 +36,9 @@ def load_arguments(self, _):
             c.argument('port', options_list=['--port', '-p'],
                        help='Port for the remote connection. Default: Random available port', type=int)
             c.argument('name', options_list=['--name', '-n'], help='Name of the webapp to connect to')
+        with self.argument_context('webapp scan') as c:
+            c.argument('name', options_list=['--name', '-n'], help='Name of the webapp to connect to')
```

Comment:
nit: Name of the webapp to scan?

Reply:
Yes

Reply:
Also looks like this is supported for Linux only - at least for now, so can you update the help to reflect this?

Reply:
That's a good point! I'll update the help section. Is there any other way to make sure users use this only on Linux based webapps?

Reply:
you can also get the site object & check if is reserved it true -> if so throw an exception saying its supported for windows only. Add this as a pre-validation before making any other API calls